### PR TITLE
test: add wave 0 verification spikes

### PR DESCRIPTION
**Pull request by harshanandak**
## Summary
Adds Wave 0 verification spike evidence and executable checks for Cursor/Codex surfaces, patch anchor stability, and cross-machine config race behavior. Also fixes Windows Git Bash path normalization for Node-backed helper scripts so the branch passes the full validation gate on this machine.

## Changes
- Added Wave 0 spike docs, evidence, and executable bench tests.
- Added `patch-anchor-stability-bench.js` and `config-race-bench.js` with threshold tests.
- Normalized Git Bash `/c/...` paths before passing script/artifact paths to Windows `node` in validation helper scripts.

## Testing
- `bun run check` exited 0.
- Full validate summary: 818 pass, 36 skip, 0 fail; quality gate reported all checks passed.
- Pre-push hook passed lint, team-sync, targeted tests, and affected edge-case tests.

## Beads
Closes forge-ymff

Notes: local Beads runtime update was blocked in this worktree by missing Dolt database `forge`; PR body links the issue for closure sync.

### CODE REVIEW

File: scripts/spikes/config-race-bench.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  };
+}
+
+function runBench(options = {}) {
+  const trials = options.trials ?? 50;
+  const threshold = options.threshold ?? 0.05;
+  const start = performance.now();
+  const results = Array.from({ length: trials }, (_, index) => mergeTrial(index + 1));
+  const manualResolves = results.filter((result) => result.manualResolve);
+  const manualResolveRate = manualResolves.length / trials;
```
[the remark this answers is not in the document]
Comment (harshanandak) [reply scripts/spikes/config-race-bench.js:75]: Fixed in commit 73a5793: config-race-bench now rejects zero trial counts before computing manualResolveRate, so the benchmark cannot divide by zero or emit invalid metrics. Added regression coverage in test/spikes/w0-verification-spikes.test.js.

File: scripts/spikes/patch-anchor-stability-bench.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    if (alias && currentAnchors.has(alias)) {
+      resolved += 1;
+      continue;
+    }
+    orphaned += 1;
+    orphanIds.push(patch.id);
+  }
+
+  const elapsedMs = performance.now() - start;
+  const orphanRate = orphaned / patchCount;
```
[the remark this answers is not in the document]
Comment (harshanandak) [reply scripts/spikes/patch-anchor-stability-bench.js:86]: Fixed in commit 73a5793: patch-anchor-stability-bench now requires patchCount to be a positive integer before computing orphanRate, preventing division by zero. Added regression coverage for zero patch counts.

File: scripts/spikes/config-race-bench.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (harshanandak) [reply scripts/spikes/config-race-bench.js:75]: Fixed in commit 73a5793: config-race-bench now validates trials with Number.isInteger and requires a positive integer, so fractional trial counts are rejected. Added regression coverage for fractional trial counts.

File: scripts/spikes/patch-anchor-stability-bench.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+const { performance } = require('node:perf_hooks');
+
+function readNumberFlag(args, name, fallback) {
+  const index = args.indexOf(name);
+  if (index === -1) return fallback;
+  const value = Number(args[index + 1]);
+  if (!Number.isFinite(value) || value < 0) {
+    throw new Error(`${name} must be a non-negative number`);
+  }
+  return value;
```
[the remark this answers is not in the document]
Comment (harshanandak) [reply scripts/spikes/patch-anchor-stability-bench.js:13]: Fixed in commit 73a5793: patch-anchor-stability-bench now validates patchCount with Number.isInteger and requires a positive integer, so fractional patch counts are rejected. Added regression coverage for fractional patch counts.

File: scripts/spikes/patch-anchor-stability-bench.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    orphanIds,
+  };
+}
+
+function main() {
+  const args = process.argv.slice(2);
+  const result = runBench({
+    anchorCount: readNumberFlag(args, '--anchors', 500),
+    patchCount: readNumberFlag(args, '--patches', 50),
+    renameCount: readNumberFlag(args, '--renames', 50),
```
[the remark this answers is not in the document]
Comment (harshanandak) [reply scripts/spikes/patch-anchor-stability-bench.js]: Fixed in commit 4173f05: the patch anchor CLI now parses --patches once and uses that parsed patchCount as the fallback for --renames when the renames flag is omitted. Added regression coverage proving `--patches 25` defaults renamedAnchors to 25 and still passes.
